pkg/operator/ceph/cluster/cluster.go

```diff
@@ -352,3 +372,49 @@ func versionSupported(version string) bool {
 	}
 	return false
 }
+
+func (c *cluster) setOrchestrationRunning(yesNo bool) {
```

Comment:
This is only called with `false`. What if we just rename it to `unsetOrchestrationRunning()`?

Reply:
Seconded. :)

Reply:
OK. Will change it accordingly!